### PR TITLE
Fix for react-native-vector icons missing or wrongly mapped

### PR DESCRIPTION
**Pull request by ajaybhatia**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "native-base",
+  "name": "@ajaybhatia/native-base",
```
Comment (sankhadeeproy007) [inline package.json]: I think you meant to add this only for your fork and not here.

File: package.json [hunk 2]
```diff
@@ -66,7 +66,7 @@
     "react-test-renderer": "16.4.1"
   },
   "gitHead": "5bbeacc403ba97622703699132c55d8359344004",
-  "homepage": "https://github.com/GeekyAnts/NativeBase#readme",
+  "homepage": "https://github.com/ajaybhatia/NativeBase#readme",
```
Comment (sankhadeeproy007) [inline package.json]: I think you meant to add this only for your fork and not here.

File: package.json [hunk 3]
```diff
@@ -77,7 +77,7 @@
   "readmeFilename": "README.md",
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/GeekyAnts/NativeBase.git"
+    "url": "git+https://github.com/ajaybhatia/NativeBase.git"
```
Comment (sankhadeeproy007) [inline package.json]: I think you meant to add this only for your fork and not here.

**Comment by shivrajkumar** [comment]
This is no longer relevant as the package is already updated and mapped.

**Pull request closed without merging**